addons/project/static/src/js/tours/project.js

```diff
@@ -118,13 +118,14 @@ registry.category("web_tour.tours").add('project_tour', {
     position: "bottom",
     run: "click",
 }, {
-    trigger: ".o_field_widget[name='user_ids'] input",
+    trigger: ".o_field_widget[name='user_ids']",
     extra_trigger: '.o_form_project_tasks',
     content: _t("Assign a responsible to your task"),
     position: "right",
     run(helpers) {
-        this.anchor.click();
-    }, 
+        this.anchor = document.querySelector(".o_field_widget[name='user_ids'] input"); //passing an HTML Element for hoot click
+        this.anchor.click()
+    }
```

Comment:
Cannot we just do that:

```suggestion
    run: "click",
```

?